packages/opentelemetry-host-metrics/package.json

```diff
@@ -59,9 +59,8 @@
     "typescript": "4.3.5"
   },
   "dependencies": {
-    "@opentelemetry/api-metrics": "^0.32.0",
-    "@opentelemetry/core": "^1.0.0",
-    "@opentelemetry/sdk-metrics": "^0.32.0",
+    "@opentelemetry/core": "^1.8.0",
```

Comment:
While checking where it is actually being used, I found that it is listed as a dependency but not imported anywhere in the package.
I think it should be removed (not sure if it should be done as part of this PR, but could be nice while we are at it)

Reply:
done